api_tests/base/test_serializers.py

```diff
@@ -215,7 +215,7 @@ def test_counts_not_included_in_link_fields_by_default(self):
         res = self.app.get(self.url)
         relationships = res.json['data']['relationships']
         for relation in relationships.values():
-            if relation == {}:
+            if relation == {} or relation == {'data': None}:
```

Comment:
Is the `relation == {}` part of this still necessary? Do you know what relationship values would be an empty dict?